Formula/apng2gif.rb

```diff
@@ -0,0 +1,20 @@
+class Apng2gif < Formula
+  desc "Convert APNG animations into animated GIF format."
+  homepage "https://apng2gif.sourceforge.io/"
+  url "https://downloads.sourceforge.net/apng2gif/apng2gif-1.8-src.zip"
+  sha256 "9a07e386017dc696573cd7bc7b46b2575c06da0bc68c3c4f1c24a4b39cdedd4d"
+
+  depends_on "libpng"
+
+  def install
+    inreplace "Makefile", "gcc", ENV.cc
```

Comment:
So, should I remove it?
This line was imitate by [typespeed.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/typespeed.rb)